### PR TITLE
FORMS-674: Search

### PR DESCRIPTION
**Pull request by rimi-itk**
https://jira.itkdev.dk/browse/FORMS-674

Improvements and clean ups before implementing search.

### CODE REVIEW

File: src/Commands/Commands.php [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+        $this->output()->writeln(Yaml::dump($data, PHP_INT_MAX));
+        break;
+
+      default:
+        throw new \InvalidArgumentException(sprintf('Unknown type: %s', $type));
+    }
+  }
+
+  /**
+   * Read person.
```
Comment (jekuaitk) [inline src/Commands/Commands.php]: Should this say `Read bruger` to avoid confusion?

File: src/Commands/Commands.php [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+   * @param string $type
+   *   The object type to search for (user, person, …).
+   * @param string $query
+   *   The search query (JSON)
+   *
+   * @command os2forms_organisation:search
+   *
+   * @usage os2forms_organisation:search --help
+   * @usage os2forms_organisation:search person '{"navntekst": "Anders And"}'
+   * @usage os2forms_organisation:search bruger '{"brugernavn": "user"}'
```
Comment (jekuaitk) [inline src/Commands/Commands.php]: Maybe we should add a comment explaining that `brugernavn` is AZ-ident?

File: src/Helper/OrganisationHelper.php [hunk 3]
```diff
@@ -174,4 +167,89 @@ public function getManagerInfo(string $userId): array {
...
+  public function searchAdresse(array $query): array {
+    /** @var \ItkDev\Serviceplatformen\Service\SF1500\AdresseService $service */
+    $service = $this->getSF1500()->getService(AdresseService::class);
+    $result = $service->soeg($query);
+
+    return $result;
+  }
+
+  /**
+   * Search for persons.
```
Comment (jekuaitk) [inline src/Helper/OrganisationHelper.php]: This should probably say `bruger`

File: src/Commands/Commands.php (hunk 1 above)
Comment (rimi-itk) [reply src/Commands/Commands.php]: Yes. I'll fix it.

File: src/Helper/OrganisationHelper.php (hunk 3 above)
Comment (rimi-itk) [reply src/Helper/OrganisationHelper.php]: Will fix.

**Review by jekuaitk — APPROVED** [review]

**Review by jekuaitk — APPROVED** [review]

**Pull request merged**